frontend/src/containers/pages/AuthoringTool/CanvasSideBar/SceneSettings.jsx

```diff
@@ -70,6 +128,54 @@ export default function SceneSettings() {
               shrink: !!currentScene?.time,
             }}
           />
+          <FormControl fullWidth>
+            <CustomInputLabel>Scene Role(s)</CustomInputLabel>
+            <Select
+              className={styles.selectInput}
+              MenuProps={{
+                getContentAnchorEl: null,
+              }}
+              value={selectedRoles}
+              onChange={(event) => setSelectedRoles(event.target.value)}
+              renderValue={(selected) =>
+                selected.length === roleList.length
+                  ? "All"
+                  : selected.join(", ")
+              }
+            >
+              {roleList && roleList.length > 0 ? (
+                <div style={{ display: "flex", flexDirection: "column" }}>
```

Comment:
Could we add some padding to the dropdown? It could make the UI look a lot nicer.
![image](https://github.com/UoaWDCC/VPS/assets/100455176/9aa6d845-9133-491f-9931-4e4fd068b6c4)
vs
![image](https://github.com/UoaWDCC/VPS/assets/100455176/ee0f2f55-be0b-46f8-86ea-2fd29ec93da2)

Reply:
(Same story with the message box  that appears when no roles are present)

Reply:
![image](https://github.com/UoaWDCC/VPS/assets/161205868/ef89a8eb-01b8-4ae1-acca-fbf0aac0aa53)